<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QWidget" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::East</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <attribute name="title">
       <string>设备管理</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="btn_add">
             <property name="whatsThis">
              <string>btn_table</string>
             </property>
             <property name="text">
              <string>添加</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/images/add64.png</normaloff>:/images/add64.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_del">
             <property name="whatsThis">
              <string>btn_table</string>
             </property>
             <property name="text">
              <string>删除</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/images/del64.png</normaloff>:/images/del64.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_apply">
             <property name="whatsThis">
              <string>btn_table</string>
             </property>
             <property name="text">
              <string>应用</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/images/apply.png</normaloff>:/images/apply.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="styleSheet">
            <string notr="true">QComboBox {
    border:none;
	background:white;
}
QComboBox::drop-down{
	border-style: none;
}

</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>其他</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
